examples/tfengine/full.hcl

```diff
@@ -281,7 +334,8 @@ data "google_secret_manager_secret_version" "db_password" {
   project = "example-secrets"
 }
 EOF
-      } */
+ */
```

Comment:
make this align with the /* TODO(user) ?

Reply:
This is done by the terragrunt hclfmt